docs/IRISMapCube.rst

```diff
@@ -0,0 +1,155 @@
+
+If you don't have a lot of RAM memory or if you are loading a huge file, we recommend to
+use the memmap kwarg. By using it, you will only load what you need to run but some
+methods that requires all the file will not be accessible. You can use the memmap
+kwarg by doing: ::
+
+    >>> from irispy.sji import read_iris_sji_level2_fits
+    >>> my_cube = read_iris_sji_level2_fits(my_fits_file, memmap=True)
+
+Class Structure
+---------------
+
+So now, ``my_cube`` is an IRISMapCube object and we can access to a lot of information like:
+
+- ``my_cube.data`` : In this attribute, we can find the data array.
+- ``my_cube.wcs`` : This attribute contains the WCS.
+- ``my_cube.mask`` : This attribute can be used to mask some data (eg. the dust particle).
+- ``my_cube.uncertainty`` : The data uncertainty is calculated as the square root of data
+  plus squared reading noise in photon unit.
+
+Dimensions
+----------
+
+As the IRISMapCube object inherits from NDCube_, we can use the two properties of NDCube_
+which allow us to the data shape and the axis types of our IRISMapCube object. These
+properties are described in the NDCube.Dimensions_ section.
+
+Cropping and Indexing
+---------------------
+
+One of the most powerful capability of IRISMapCube, coming from NDCube_, is the slicing
+process. To slice the cube, we can slice the IRISMapCube with an Array-like Indexing or
+we can crop it by the Real World Coordinates. As the IRISMapCube object inherits from
+NDCube_, we can use the described processes in the NDCube.Slicing_ section.
+
+Data Manipulation
+-----------------
+
+Now we have our IRISMapCube object and we know how access to all the information it contains,
+we can manipulate the data with the below presented methods.
+
+Exposure Time Correction
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We can apply the exposure time correction to data and to the uncertainty and
+this method adjusts the unit: ::
+
+    >>> my_cube.apply_exposure_time_correction()
+
+We also can undo the exposure time correction by doing: ::
+
+    >>> my_cube.apply_exposure_time_correction(undo=True)
+
+The correction is only applied (undone) if the object's unit doesn't (does) already
+include inverse time. This can be overridden so that correction is applied (undone)
+regardless of unit by setting ``force=True``. Use one of the two lines above to apply
+(undone) by using the force kwarg: ::
+
+    >>> my_cube.apply_exposure_time_correction(force=True)
+    >>> my_cube.apply_exposure_time_correction(undo=True, force=True)
+
+Dust particle mask
+^^^^^^^^^^^^^^^^^^
+
+If we take time to look inside the data, we can see that some spots are not relevant with
+the data we want to study. These spots are dust particles that can come from space or a
+not perfect construction of the satellite. In IRISMapCube, we can use a method that will
+modify the mask of our object with the dust particles positions. We can use this method
+by doing: ::
```

Comment:
Suggested rewrite: 

...inside the data, we see that some pixels are obscured by dust, and so do not reflect the emission from the Sun at that location.  The apply_dust_mask method of IRISMapCube can be used to add the locations of the dust pixels to the mask so that we can easily exclude them fro our analysis. 

>>> my_cube.apply_dust_mask()

Reply:
Yes this is better, my sentences was horrible there !